apps/dashboard/src/lib/components/Org/PathWay/PathwayCard.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import { t } from '$lib/utils/functions/translations';
+  import { ImageLoader, SkeletonPlaceholder } from 'carbon-components-svelte';
+
+  export let data = {
+    title: 'Become a Professional UX designer',
+    description:
+      'This is a full pathway to become a professional UX designer,with practical examples',
+    avatar_url: '',
```

Comment:
This should be `banner_url` looking at the API.

Also please pass them as individual props instead of passing an object as the prop

Reply:
alright